### ISSUE TITLE
Force old youtube design

### ISSUE THREAD

**Issue by zenwarr** [YePpHa/YouTubeCenter#2444]
As all you know, YouTube has recently switched to the new design which completely broke this extension. I prefer to stay on the old design while it is still available. Hopefully, you can switch back to the old design by choosing "Restore old YouTube" menu option from the settings menu. But there is a serious issue (i do not know if other people face it too, but it is annoying for me). Even if you switch to the old design, YouTube keeps switching you back to the new design from time to time (and you have to disable YTC, reload page, click "Restore old YouTube" again, wait until page is reloaded, enable YTC back and reload page again; all this is very annoying). I do not know what is the cause for it. But it happens several times a day (for me, at least).

But there is a solution. You can install EditThisCookie extension which allows you to make a cookie read-only and forbid changing its value by a site. To prevent automatic switching back to the old design you should open youtube.com, switch back the old design, open EditThisCookie popup, find a cookie named "PREF" and click on lock icon near this cookie. So YouTube will not be able to switch you back to the new design. (If you cannot return to the old design from menu, edit `PREF` cookie and set f6 option to value 8, so your cookie should look like `f1=50000000&f6=8&f5=30` and then reload the page). Hope it helps somebody.

It would be great if this feature could be integrated into YouTubeCenter, as it breaks the new design anyway.

**Comment by Denebulas** [comment]
Thank you I have the same issue.

**Comment by DuskyVoltage** [comment]
I tried what you described, @Zenwarr, but I couldn't get EditThisCookie to actually display any cookies at all?
Any help?

![capture](https://user-images.githubusercontent.com/9156437/30245736-6f1386ce-95db-11e7-91d9-9e2017dd1200.JPG)

**Comment by Denebulas** [comment]
I used the chrome  extension cookies.

**Comment by rikkdekkard** [comment]
You guys could try using any of these two userscripts:

[https://greasyfork.org/en/scripts/32906-get-me-old-youtube](url)
or
[https://greasyfork.org/en/scripts/32759-old-youtube-layout](url)

I'm not sure how well they work though, as I didn't personally have to use them (yet).

I also think (with YTC  doomed anyway) it's best if we let YePpHa focus on maia-yt as the man seems busy enough as it is.

**Comment by rayman89** [comment]
Just go to youtube.com and on the top right corner press the menu button and at the end youll find an item that says "enable old youtube layout" or something like that.

**Comment by DuskyVoltage** [comment]
@rayman89 Yes, that's known. The problem with that solution is Youtube frequently reverts the change, sometimes multiple times a day. It's a nuisance.

**Comment by rayman89** [comment]
Great. Maybe is because you delete your data or something because I changed it once it's been like that for a loooooooooong time.

**Comment by Watilin** [comment]
I’ve set an addon (Self-Destructing Cookies for the curious) to auto-delete Youtube cookies when I leave the site. So, having a script that automatically reverts to old Youtube is necessary to me. And I doubt I’m the only one in this situation.

The “Get Me Old Youtube” userscript @rikkdekkard linked to does the job perfectly. When you look at its code you can tell that it simply performs some PREF cookie reading and modifies it if needed.

**Comment by marktedrow1** [comment]
I agree with the New YouTube design sucking. These companies always want to take what works and screw it up instead of just leaving it alone. I had the same problem with the new Firefox Quantum. I have used Firefox for 20 years as well as bragging that they had the best add-ons anywhere, but the update disabled alot those  add-ons that I use, so I had to uninstall it on both my Pc and Cell and reinstall the previous version and disable the automatic updates. Alot of people have done this but these companies like Mozilla and YouTube don't really pay attention to what users want.

**Comment by cr4zyd3v** [comment]
Many many many thanks my friend!
